docs/webhooks.rst

```diff
@@ -1,78 +1,128 @@
 Webhooks
 ========
 
-Webhooks are pretty amazing, and help to turn the web into a push instead of
-pull platform. We have support for hitting a URL whenever you commit to your
-project and we will try and rebuild your docs. This only rebuilds them if
-something has changed, so it is cheap on the server side. As anyone who has
-worked with push knows, pushing a doc update to your repo and watching it get
-updated within seconds is an awesome feeling.
+The primary method that Read the Docs uses to detect changes to your
+documentation is through the use of *webhooks*. Webhooks are configured with
+your repository provider and with each commit, merge, or other change to your
+repository, Read the Docs is notified. When this happens, we determine if the
+change is related to an active version for your project, and if it is, a build
+is triggered for that version.
+
+Webhook creation
+----------------
+
+If you import a project using a connected account, a webhook will be set up
+automatically for your repository. However, if your project was not imported
+through a connected account, you may need to manually configure a webhook for
+your project.
+
+To manually set up a webhook, you must first create the corresponding
+*integration* on your project's **Integrations** admin page. On this page, you
+can click **Add integration** and select the integration type you'd like to add.
+In order to set up a new webhook with your provider, you'll need the *webhook
+URL*. You can get this URL by selecting the integration you just added on the
+list of integrations.
```

Comment:
`You can get this URL by selecting the integration you just added on the
 list of integrations.` sounds confusing, and could be more explicit. Should we be returning them to the detail page, instead of the listing, when they add a new integration?